starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,54 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="C:\Users\fredrik\Downloads\lab-css-recipes-clone-master\lab-css-recipes-clone-master\starter-code\apple-pie-starter-code\css\main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+  <div id="applepieimg">
+<h1 id="applepieh1">Apple Pie</h1>
+  </div>
+<p class="grandma">This was my grandmother's apple pie recipe. I have never seen another one
 quite like it. It will always be my favorite and has won me several first place
 prizes in local competitions. I hope it becomes one of your favorites as well!
+</p>
```

Comment:
Take a bit more care with the indentation:
```
<body>
  <div id="applepieimg">
    <h1 id="applepieh1">Apple Pie</h1>
  </div>
  <p class="grandma"> This was my grandmother 's apple pie recipe. I have never seen another one quite like it.It will always be my favorite
    and has won me several first place prizes in local competitions.I hope it becomes one of your favorites as well!
  </p>
...
```